### PR TITLE
doc/md: add doc for notifications in schema monitoring 

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

File: doc/md/monitoring/07-notifications.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: Notifications
+---
+
+To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.
+Notifications can be sent to Slack, Workplace, or a plain webhook URL.
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:6]:
```suggestion
Notifications can be sent to Slack, Workplace, or any HTTP endpoint.
```

File: doc/md/monitoring/07-notifications.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+#### Setting up Notifications
+
+To set up notifications, go to the **webhooks** tab of the database you want to monitor and click on the **Add Webhook** button.
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:11]: 1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.
2. Click on "Monitoring" in the top-level navigation on the left side of the screen. 
3. Choose ..

File: doc/md/monitoring/07-notifications.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: Notifications
+---
+
+To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.
+Notifications can be sent to Slack, Workplace, or a plain webhook URL.
+
+
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:8]:
```suggestion

```
Remove double lines in the rest of the doc as well please.

File: doc/md/monitoring/07-notifications.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Choose where you want to send the notification, and fill in the required information.
+
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/monitoring/webhook-create.png)
+</details>
+
+
+You can send ping webhooks to test the connection and verify that the notification is working as expected by clicking on the **Send Test Notification** button.
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:22]: To test your configuration, use the ...

File: doc/md/monitoring/07-notifications.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/monitoring/send-ping.png)
+</details>
+
+
+After setting up the notification, you will receive notifications whenever there are a schema changes in the database.
+
+Lest create new table in our database and see how the notification works.
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:33]: Lest -> Lets

This is a new section. Needs missing subheader

File: doc/md/monitoring/07-notifications.mdx [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
+title: Notifications
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:2]: are we calling this "webhooks" or "notifications" or "subscriptions"? we need to be consistent in the UI, docs, and marketing

File: doc/md/monitoring/07-notifications.mdx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.
+
+2. Click on **Monitoring** in the top-level navigation on the left side of the screen.
+
+3. Choose the database you want to monitor and click on the **Webhooks** tab.
+
+4. Click on the **Add Webhook** button and fill in the required information.
+
+<details>
```
Comment (rotemtam) [inline doc/md/monitoring/07-notifications.mdx:18]: 5. Select the integration type

Add a tabs component for each type documenting the fields in the form and their meaning.

File: doc/md/monitoring/07-notifications.mdx (hunk 6 above)
Comment (ronenlu) [reply doc/md/monitoring/07-notifications.mdx:2]: I think it is best to go with webhooks, I will change it to be consistent

**Review by rotemtam — APPROVED** [review]

**Pull request merged**